_rules/SC2-4-2-page-has-title.md

```diff
@@ -34,7 +34,7 @@ The page contains at least one `title` element.
 
 ### Expectation 2
 
-The first `title` element contains [non-empty text](#non-empty).
+The first `title` element contains [text nodes](https://www.w3.org/TR/dom/#text) that do not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
```

Comment:
"test nodes that do not only consist of unicode separator characters" is far from plain language. Please have a look at some of my previous suggestions for ideas.